src/dmd/attrib.d

```diff
@@ -1,10 +1,12 @@
 /**
- * Defines declarations of various 'attributes'.
- * The term attribute is used very loosly, since it includes:
+ * Defines declarations of various attributes.
+ *
+ * The term 'attribute' refers to things that apply to a larger scope than a single declaration.
```

Comment:
```suggestion
 * The term 'attribute' refers to things that can apply to a larger scope than a single declaration.
```

E.g. depending on how you declare it, you might have `deprecated` as an attribute which is a parent of Dsymbols, or directly as part of the Dsymbol's storage classes.